modules/commands/list.json

```diff
@@ -500,6 +518,56 @@
       }
     ]
   },
+  {
+    "name": "ssl",
+    "aliases": [
+      "usessl"
+    ],
+    "title": "The useSSL error!",
+    "description": "The useSSL error appears when you have not added the correct lines in the right place in the configuration. Make sure you do not uncomment the ones already there, but rather add them underneath the `characterEncoding` line.",
+    "fields": [
+      {
+        "key": "Read me!",
+        "value": "https://github.com/lucko/LuckPerms/wiki/Storage-system-errors#mysql-ssl-errors"
+      },
+      {
+        "key": "What is SSL?",
+        "value": "https://www.cloudflare.com/learning/ssl/what-is-ssl/"
+      }
+    ]
+  },
+  {
+    "name": "timeout",
+    "aliases": [
+      "connectiontimeout"
+    ],
+    "title": "The connectionTimeout error!",
+    "description": "The useSSL error appears when you have not added the correct lines in the right place in the configuration. Make sure you do not uncomment the ones already there, but rather add them underneath the `characterEncoding` line.",
+    "fields": [
+      {
+        "key": "Read me!",
+        "value": "https://github.com/lucko/LuckPerms/wiki/Storage-system-errors#mysql-ssl-errors"
+      },
+      {
+        "key": "What is SSL?",
+        "value": "https://www.cloudflare.com/learning/ssl/what-is-ssl/"
+      }
+    ]
+  },
+  {
+    "name": "hosting",

```

Comment:
Sorry, I think this is out of scope for a bot command.

Reply:
This was an artefact I thought I had removed, haha. Will fix when I get back to the PC

Reply:
removed